backend/src/routes/api/config/index.ts

```diff
@@ -15,7 +15,14 @@ module.exports = async (fastify: KubeFastifyInstance) => {
   fastify.patch(
     '/',
     secureAdminRoute(fastify)(async (request: FastifyRequest, reply: FastifyReply) => {
-      reply.send(setDashboardConfig(fastify, request.body));
+      if (
+        typeof request === 'object' &&
+        'body' in request &&
+        request.body &&
+        typeof request.body === 'object'
+      ) {
+        reply.send(setDashboardConfig(fastify, request.body));
+      }
```

Comment:
Without it, it errors:. I presume it's because it isn't defined in the `FastifyRequest` interface

```
Argument of type 'unknown' is not assignable to parameter of type 'RecursivePartial<DashboardConfig>'.ts(2345)
```

Reply:
Ok i can understand with `request.body` having type `unknown` but why would `request` possibly be `null` here?